Kwenta/kwenta
# Hotfix: Downgraded the rainbokw to v0.5 and Added a max query limit

## Description
There are two prod issues
1. The latest version (v0.5.1) of the rainbowkit doesn't work with the metamask mobile (got the same error with the offical example https://www.rainbowkit.com/).  Already submitted a bug report https://github.com/rainbow-me/rainbowkit/issues/728
2. Leaderboard data is missing because of the query is over the limit on the subgraph

Short-term solution:
1. Downgrade the rainbowkit to v0.5
2. Add a query limit to the `useGetStats`


## Related issue


## Motivation and Context
Make the PROD work again

## How Has This Been Tested?
1. Open the dapp from mobile metamask and the wallet is connected normally
2. Go to the leaderboard page and see the data

## Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/4819006/189306527-7ea33cbb-6f33-4bcd-b8d7-0cbf6e4444ac.png)
![image](https://user-images.githubusercontent.com/4819006/189306667-0df40d2e-d6bf-4d80-b889-82fdfd28b8bd.png)
